components/nav/Navbar.vue

```diff
@@ -21,13 +21,13 @@
         id="navbarSupportedContent"
       >
         <ul class="navbar-nav ms-auto">
-          <li class="nav-item me-2" @click="collapseNav">
-            <NuxtLink to="/" class="nav-link">
-              Home
-              <HoverSvg />
```

Comment:
Why did we removed the `<HoverSvg />`

Reply:
Yes, I had this in mind. Removing it now.